<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerdialog</class>
 <widget class="QDialog" name="registerdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/lo.png</normaloff>:/img/lo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color:#00aeff;
}</string>
  </property>
  <widget class="QLabel" name="comment_register">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#comment_register{
	
	background-color: rgb(255, 255, 255);
	color:black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="status_register">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Demi</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#status_register{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Статус</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>91</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="name_register">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#name_register{
	background-color: rgb(247, 247, 255);
	color:black;
	border-radius: 15px;
}</string>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="birthDate">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#birthDate{
	background-color: rgb(247, 247, 255);
	color:black;
	border-radius: 15px;
}</string>
      </property>
      <property name="text">
       <string>Birthdate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="email_register">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#email_register{
	background-color: rgb(247, 247, 255);
	color:black;
	border-radius: 15px;
}</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="address_register">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#address_register{
	background-color: rgb(247, 247, 255);
	color:black;
	border-radius: 15px;
}</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusCombo">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#statusCombo{
	background-color: rgb(247, 247, 255);
	color:black;
	border-radius: 15px;
}</string>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="password_regi">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#password_regi{
	background-color: rgb(247, 247, 255);
	color:black;
	border-radius: 15px;
}</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="nameedit_register">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>339</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#nameedit_register{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="birthDateedit_register">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#birthDateedit_register{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailedit_register">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>339</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#emailedit_register{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressedit_register">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#addressedit_register{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enter_register">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#enter_register{
	background-color: rgb(85, 85, 255);
	border-radius: 15px;
	color:white;
}
#enter_register:hover{
	background-color: rgb(255, 85, 0);
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_register">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#comboBox_register{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <item>
    <property name="text">
     <string>A - subscription only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>R - only reading hall</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B - reading hall and subsccription</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxDay_register">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#comboBoxDay_register{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMonth_register">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#comboBoxMonth_register{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthDate_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Amiri</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#birthDate{
	background-color: rgb(247, 247, 255);
	color:black;
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthDate_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Amiri</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#birthDate{
	background-color: rgb(247, 247, 255);
	color:black;
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthDate_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Amiri</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#birthDate{
	background-color: rgb(247, 247, 255);
	color:black;
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordedit_register">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#passwordedit_register{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_regi">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>700</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#exit_regi{
	
	
	background-color: rgb(85, 85, 127);
	border-radius: 15px;
	color:white;
}
#exit_regi:hover{
	background-color: rgb(255, 85, 0);
}</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QLabel" name="getpersonal_data_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>680</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#getpersonal_data_2{
	
	background-color: rgb(255, 255, 255);
	color:black;
}</string>
   </property>
   <property name="text">
    <string>Примечание: Запомните свой пароль и имя пользователя,&lt;br&gt; которые появятся здесь</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>250</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>320</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#graphicsView{
	background-image: url(:/img/lieson.png);
	border-color: rgb(0, 174, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>47</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>110</y>
     <width>31</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="titlegraph">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>320</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#titlegraph{
	
	background-image: url(:/img/imag.png);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3{
	background-color: qlineargradient(spread:pad, x1:0.096, y1:0.46, x2:0, y2:0, stop:0 rgba(0, 174, 255, 255), stop:1 rgba(226, 170, 255, 255));
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3{
	background-color: qlineargradient(spread:pad, x1:0.096, y1:0.46, x2:0, y2:0, stop:0 rgba(0, 174, 255, 255), stop:1 rgba(226, 170, 255, 255));
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3{
	background-color: qlineargradient(spread:pad, x1:0.096, y1:0.46, x2:0, y2:0, stop:0 rgba(0, 174, 255, 255), stop:1 rgba(226, 170, 255, 255));
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
